pkg/executor/terraform/terraform.go

```diff
@@ -53,12 +70,16 @@ func Plan(destroy bool) (string, error) {
 		args = append(args, "-destroy")
 	}
 
-	output, err := terraform(context.Background(), os.Environ(), args...)
+	var cmd = shell.Command{
+                Command: "terraform",
+                Args:    args,
+        }
+	output, err := shell.Execute(cmd)
 	if err != nil {
 		return "", err
 	}
 
-	return combineOutput(output), nil
+	return output, nil
 }
 
 func combineOutput(in []string) string {
```

Comment:
Looks like `combineOutput()` is not being used anywhere anymore, should it be deleted?

Reply:
Yeah.

Reply:
done